NineMensMorris/src/piece.cpp

```diff
@@ -57,17 +60,26 @@ void Piece::leaveEvent(QEvent */*event*/) {
     update();
 }
 
+void Piece::movePiece(Space *space) {
+/* Animates a piece's movement */
+    pieceAnimation->setDuration(500);
+    pieceAnimation->setStartValue(this->geometry());
+    pieceAnimation->setEndValue(QRect(space->x() - 5, space->y() - 5, 31, 31));
+    pieceAnimation->start(QAbstractAnimation::DeleteWhenStopped);
+    QTimer::singleShot(5000, this, SLOT(frick()));
```

Comment:
If frick() is a library signal, I couldn't find any documentation on it. Did you mean to leave this in here?

Reply:
Oh I see, you just want it to do nothing

Reply:
this was not meant to be in this PR. I have removed it.